Week10NetworkingCodingExercise/Week10NetworkingCodingExercise/Program.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.IO;
+using System.IO.Compression;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace StreamsIONetworking
+{
+    /// <summary>
+    /// Program runs calculations for querty string.
+    /// </summary>
+    class Program
+    {
+        /// <summary>
+        /// Calls the methods that do the calculations.
+        /// </summary>
+        /// <param name="args"></param>
+        static void Main(string[] args)
+        {
+            //UriDemo();
+            //HttpListenerDemo();
```

Comment:
Going forward, it would be good to at least have something functional even if not fully correct when you submit your assignment for grading. As it is, the program didn't do anything when run because both of these lines were commented out. (-2.5)